src/locales/es/translations/actions/index.ts

```diff
@@ -0,0 +1,15 @@
+export default {
+  actions: {
+    label: 'Acciones',
+    edit: 'Editar',
+    save: 'Guardar',
+    update: 'Actualizar',
+    complete: 'Finalizar',
+    delete: 'Borrar',
+    cancel: 'Cancelar',
+    new: 'Nuevo',
+    list: 'Enumerar',
+    search: 'Buscar',
+    confirmDelete: 'Borrar Confirmación',
```

Comment:
Not sure how many translations you want to add, but looking strictly at the English translations, it appears as though you are missing a couple within `actions/index.ts`. Specifically, look at:

- next
- previous
- page
- add
- view
- logout